base/db/R/dbfiles.R

```diff
@@ -43,10 +43,10 @@ dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate,
                                 parentid = NA, con, hostname = PEcAn.remote::fqdn(), allow.conflicting.dates = FALSE, ens = FALSE) {
   name <- basename(in.path)
   hostname <- default_hostname(hostname)
-
+  
```

Comment:
Looks like all the changes to this file are adding trailing whitespace -- can it be dropped from the PR?